src/popup/config/config.json

```diff
@@ -77,9 +82,19 @@
           "label": "Dogfood",
           "host": "df-devicemanagement.onecloud.azure-test.net"
         },
+        {
+          "label": "Int",
+          "host": "intune.microsoft.com",
+          "params": {
+            "feature.canmodifystamps": "true",
+            "feature.canmodifyextensions": "true",
+            "feature.intune_env": "selfhost",
+            "feature.CloudPCGraphVersion": "testprodbeta_cpc_int"
+          }
+        },
         {
           "label": "Selfhost",
-          "host": "endpoint.microsoft.com",
+          "host": "intune.microsoft.com",
```

Comment:
@PxlBuzzard Currently Intune portal changed to intune.microsoft.com so update this as well